src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/HealthCertificateValidationService.swift

```diff
@@ -62,15 +64,34 @@ final class HealthCertificateValidationService: HealthCertificateValidationProvi
 	) {
 		
 		// 1. Apply technical validation
-		let expirationDate = healthCertificate.cborWebTokenHeader.expirationTime
 		
+		let signatureInvalid: Bool
+		var signatureValidationError: Error?
+		// DONT FORGET: pass singatures array into this function!
+		let result = signatureVerifying.verify(certificate: healthCertificate.base45, with: [], and: validationClock)
```

Comment:
This is now available. You need to inject `dscListProvider: DSCListProviding` and can use it like this then:

```suggestion
		let result = signatureVerifying.verify(
			certificate: healthCertificate.base45, 
			with: dscListProvider.signingCertificates.value,
			and: validationClock
		)
```

Reply:
Done